GFX Font fix for newer GFX Font format

KurtE opened the pull request:
@wwatson4506 @mjs513 

Problem was the definitions were also still in Registers.h file... So we never actually did anything of getting the real GFX font file header file.

Does not fix rotation issues...

KurtE committed:
GFX Font fix for newer GFX Font format

Problem was the definitions were also still in Registers.h file... So we never actually did anything of getting the real GFX font file header file.

Does not fix rotation issues...

mjs513 commented:
Ok - think i know what may be the problem with screen rotation.  Will work on that when I get back from shopping

KurtE committed:
Update ILI_Ada_fontTest4 - system font only works rotation 0

Update the test case that in other rotations, we don't use system font.

Instead show sometext in Chancery_24_italic the name system font and warn the user that it only works in Rotation 0...
